components/server/src/main/java/com/hotels/styx/server/routing/routes/ProxyToBackendRoute.java

```diff
@@ -42,6 +42,7 @@ public static ProxyToBackendRoute proxyToBackend(HttpClient client) {
     @Override
     public StyxObservable<HttpResponse> handle(HttpRequest request, HttpInterceptor.Context context) {
         // TODO: Mikko: Styx 2.0 API:
+        // Unclear comment
         return fromRxObservable(client.sendRequest(request));
```

Comment:
The comment was there to remind me that the client is still using bare rx observables. It can be removed now as the client interface is a well-understood limitation in the 1.0 API.